docs/mdbook/default.nix

```diff
@@ -243,8 +265,10 @@ let
     substituteInPlace ./SUMMARY.md \
       --replace-fail "@NIXVIM_OPTIONS@" "$(cat ${pkgs.writeText "nixvim-options-summary.md" mdbook.nixvimOptions})"
 
-    substituteInPlace ./modules/hm.md \
-      --replace-fail "@HM_OPTIONS@" "$(cat ${mkMDDoc hmOptions})"
+    substituteInPlace ./modules/wrapper-options.md \
+      --replace-fail "@NIXOS_OPTIONS@" "$(cat ${mkMDDoc wrapperOptions.nixos})" \
+      --replace-fail "@HM_OPTIONS@" "$(cat ${mkMDDoc wrapperOptions.hm})" \
+      --replace-fail "@DARWIN_OPTIONS@" "$(cat ${mkMDDoc wrapperOptions.darwin})"
```

Comment:
Maybe we should be creating a separate file for each wrapper here, rather than substituting them into a single template file...

Something like
```nix
foldAttrs (
  str: name: options:
  str + "\n install ${mkMDDoc options} ./modules/${name}.md"
) "" wrapperOptions
```

Reply:
Up to you if this blocks the PR

Reply:
Do you want to just split them out to be separate documentation pages instead of being shown on a single page? It might be cleaner than having them smushed together on the same page right now with the formatting.

Reply:
Yeah. I think we should do it this way, and perhaps also set a `meta.docs` in each module too if we want a pre-ample header.

Perhaps that should follow in another PR to keep this one simple.

Reply:
It's kinda hard at a glance to see where one module beings and another ends with the current page. I agree we could do it cleaner but this at least shows all the different available.